### DISCUSSION TITLE
Call for distro maintainers

### DISCUSSION THREAD

**Discussion by crummel** [dotnet/source-build#2803] [Announcements]
Hi everyone, we're working on creating a GitHub team that we can tag instead of having to tag you all individually.  This will be used both here in source-build for announcements, etc, and in other repos in the dotnet org when we think an issue or PR might be of interest to you as distro or package maintainers.

Due to how GitHub's team system works, we'll need to invite you to the dotnet org first and then you'll need to accept that invitation.  We don't personally have access to invite people to the org so we wanted to batch up the first group of volunteers - please reply here if you'd like to be included.

cc @alucryd @asbjornu @ayakael @cho-m @Livius90 @mirespace @omajid

**Comment by ayakael** [comment]
Feel free to invite me `Antoine Martin (ayakael) <dev@ayakael.net>`

**Reply by alucryd** [reply]
I'm in too, Maxime Gauduin, aka alucryd, Arch Linux dev and maintainer of the various .NET packages.

**Reply by omajid** [reply]
I am in, too!

Edit: I am one of the maintainers of .NET packages on Fedora and RHEL

**Reply by mirespace** [reply]
Thank you so much! I would like to be included, of course :)

**Reply by asbjornu** [reply]
I'm in! 🚀

**Comment by crummel** [comment]
You should hopefully all have invitations to respond to now!  Welcome to the team  :)

**Comment by corngood** [comment]
Hi, I'm not sure if this is still the place for this, but I'm attempting to maintain source builds in nixpkgs.  I think it would be useful to be in this group.  Thanks!

**Reply by MichaelSimons** [reply]
Hi David, I sent you an invitation.  Glad to have you onboard.  Feel free to open PRs that add nixpkgs to https://github.com/dotnet/source-build/?tab=readme-ov-file#net-in-linux-distributions and https://github.com/dotnet/core/blob/main/linux.md#package-archives
